### ISSUE TITLE
GRNsight should read and retain all network sheets

### ISSUE THREAD

**Issue by dondi** [dondi/GRNsight#917]
GRNsight currently prioritizes network optimized sheet data and discards network if given the choice. However this interferes with Excel export and so, even if it does not use the other network data, it should retain it in the workbook object.

**Comment by Onariaginosa** [comment]
- I changed the network structure to retain all network sheets. 

- I restructured all tests to account for this change. 

- I restructured the spreadsheet controller to account for this change, while retaining usual import functionality. 
  - The only issue I have with this, is that in order to ensure that everything worked properly, I set the workbook to the preferred networks values, because that is how we currently get all of the gene and link information to render the graph. Then I attatched all network files, including the one we cloned, to the workbook object. I did this, so that I could export all network files, using their keys. 
  - I don't like this because it currently stores the chosen network twice instead of once. 
  - One way to fix this is to make the graph renderer use the information in the attached network objects, rather than the information directly in the workbook object. 
  - Another (albiet sus) solution is to keep the chosen network information as is, and only attach the networks not chosen. Then in the export process, check whether it is unweighted or weighted to determine what type of network sheet it is, then export all of the other network sheets by key.

- I changed the demo files to account for the new export mechanism (They are currently using the duplicate information method, and I will change this as soon as a preferred choice has been made.
